pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "documentation"
-requires-python = ">=3.8<3.9"
+requires-python = ">=3.9<3.10"
```

Comment:
Does ~=3.9 work here? I've never actually tried pip-compatible version specifiers for this argument, but it might do?

IIRC I added this to try and get dependabot to pick up a version without using .python-version, so the `project` key can probably go anyway.

Reply:
> Does ~=3.9 work here?

Yes, it should do: https://peps.python.org/pep-0508/#grammar

But I've ditched `[project]` as suggested. We're not building a distribution from this project as such.

And the current requirements files and workflows imply the currently used version.